src/main/java/io/javalin/core/JavalinConfig.java

```diff
@@ -34,6 +34,7 @@
 // @formatter:off
 public class JavalinConfig {
 
+    public boolean autoLowercaseUrl = false;
```

Comment:
Usually libraries call it caseInsensitiveUrl or something like that, would be valuable to name it like that (in case users search for this).